File: backend-aplicacao/Sinqia/Sinqia.Domain/Entites/Produto.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Sinqia.Domain.Entites
{
    [Table("Produto")]
    public class Produto
    {
        public Produto()
        {
            ListaProdutosCosif = new HashSet<ProdutoCosif>();             
        }

        [Key]
        public string Cod_Produto { get; set; }
        public string Des_Produto { get; set; }
        public string Sta_Status { get; set; }
        
        public virtual ICollection<ProdutoCosif> ListaProdutosCosif { get; set; }
         
    }
}

File: apresentacao/sinqia/src/app/produto/produto.ts
import { ProdutoCosif } from "./produto-cosif/produto-cosif";

export class Produto
{
  Cod_Produto : string;
  Des_Produto : string;
  Sta_Status  : string;
  ListaProdutosCosif : Array<ProdutoCosif>;

}
